lightning/src/ln/channel.rs

```diff
@@ -8466,7 +8918,11 @@ mod tests {
 		let config = UserConfig::default();
 		let features = channelmanager::provided_init_features(&config);
 		let outbound_chan = OutboundV1Channel::<&TestKeysInterface>::new(&feeest, &&keys_provider, &&keys_provider, node_b_node_id, &features, 10000000, 100000, 42, &config, 0, 42, None).unwrap();
-		let mut chan = Channel { context: outbound_chan.context };
+		let mut chan = Channel {
+			context: outbound_chan.context,
+			#[cfg(dual_funding)]
+			dual_funding_channel_context: None,
+		};
```

Comment:
This construct appears a few times, maybe it could be moved to a Channel::default(context) constructor.